<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="visualiser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1001</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>10</y>
      <width>171</width>
      <height>336</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Interpolation Sort</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sortBtn">
       <property name="text">
        <string>Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <fontweight>DemiBold</fontweight>
        </font>
       </property>
       <property name="text">
        <string>Elapsed time (sec):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeLbl">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Binary Search</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="searchBox">
       <property name="maximum">
        <number>199</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchBtn">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>350</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="indexLbl">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Index (I):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Value (V):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Size (S):</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>I^S mod V</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="valLbl">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="sizeLbl">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="resultLbl">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
